dist/tripster/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Tripster</title>
+  <base href="/">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Poppins:500&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="styles.05fa8cdd29c17451b34c.css"></head>
+<body>
+  <app-root></app-root>
+<script src="runtime-es2015.58ee69ced0f4ed992790.js" type="module"></script><script src="runtime-es5.58ee69ced0f4ed992790.js" nomodule defer></script><script src="polyfills-es5.2b9ce34c123ac007a8ba.js" nomodule defer></script><script src="polyfills-es2015.0ef207fb7b4761464817.js" type="module"></script><script src="main-es2015.1b8a40fe5bcdc673f127.js" type="module"></script><script src="main-es5.1b8a40fe5bcdc673f127.js" nomodule defer></script></body>
```

Comment:
why is this not the right way? Because if we use hash, then the browser won't cache the JS files and the user doesn't have to clear cache every time to render the new JS.